dev/scripts/env.production

```diff
@@ -30,3 +30,6 @@ WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL=
 WATCHTOWER_NOTIFICATIONS=
 WATCHTOWER_POLL_INTERVAL=30
 WATCHTOWER_REMOVE_VOLUMES=true
+PROSOPO_ADMIN_ADDRESS=5EjTA28bKSbFPPyMbUjNtArxyqjwq38r1BapVmLZShaqEedV
```

Comment:
Is this needed in production?

Reply:
not sure, setup of e2es in gha is difficult to replicate locally. Will drop and see if passing